Makefile

```
@@ -8,7 +8,7 @@ GOLANGCI_LINT := golangci-lint run --disable-all \
 	-E staticcheck \
 	-E typecheck \
 	-E unused
-VERSION := 0.6.0
+VERSION := 0.7.0
```

💄 While in beta (`0.*`), I would suggest keeping all backward compatible smallish additions just in patch releases.

```suggestion
VERSION := 0.6.1
```

---

Sure I'll update, thanks